guide-redaction/index.md

```diff
@@ -234,38 +234,40 @@ title: "Guide de rédaction du contenu du site Canada.ca"
       <li>d'orienter la personne vers les renseignements dont elle a besoin pour accomplir une tâche;</li>
       <li>de placer les renseignements complémentaires à la fin du texte.</li>
     </ul>
-    <figure class="mrgn-bttm-lg">
-      <figcaption class="text-center"> <strong>Figure 1</strong> </figcaption>
-      <img alt="Pyramide inversée illustrant comment créer une page Web en indiquant tout d'abord les renseignements clés." class="img-responsive center-block" id="fig1" src="/guide-redaction/images/fig-01-fra.jpg">
-      <details class="brdr-tp brdr-rght brdr-bttm brdr-lft">
-        <summary class="wb-toggle" data-toggle="'{&quot;print&quot;:&quot;on&quot;}'">Figure 1 - Version textuelle</summary>
-        <p>Pyramide inversée illustrant comment créer une page Web en indiquant tout d'abord les renseignements clés.</p>
+    <div class="row mrgn-tp-lg mrgn-bttm-md">
+      <div class="col-md-8">
+        <figure><img alt="Pyramide inversée illustrant comment créer une page Web en indiquant tout d'abord les renseignements clés." class="img-responsive" id="fig1" src="/guide-redaction/images/pyramide-fr.png" />
+          <figcaption>
+            <p><strong>Figure 1:</strong> pyramide inversée pour l’organisation du contenu</p>
```

Comment:
This should have a space between 'Figure 1' and the colon. So: Figure 1 : pyramide....

Reply:
Done